plentyofpets/lib/screens/admin_profile.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+// Class to display Admin account information
+class AdminProfile extends StatelessWidget {
+  const AdminProfile({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    CollectionReference users = FirebaseFirestore.instance.collection('users');
+
+    return Container(
+      margin: const EdgeInsets.all(10.0),
+      child: FutureBuilder<DocumentSnapshot>(
+        future: users.doc('JimXAqx3qgxONKm6J4Cg').get(),
+        builder:
+            (BuildContext context, AsyncSnapshot<DocumentSnapshot> snapshot) {
+          if (snapshot.hasError) {
+            return Text("Something went wrong: ${snapshot.error}");
+          }
+
+          if (snapshot.hasData && !snapshot.data!.exists) {
+            return const Text("Document does not exist");
+          }
+
+          if (snapshot.connectionState == ConnectionState.done) {
+            Map<String, dynamic> data =
+                snapshot.data!.data() as Map<String, dynamic>;
+
+            return Padding(
+              padding: const EdgeInsets.fromLTRB(20, 40, 20, 0),
+              child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text('Name: ${data['firstName']}' ' ' + data['lastName']),
```

Comment:
This widget looks, good, but I think you are missing a plus sign in here as well as line 36? Also might be worth considering just making this all a template and not using concatenation -
`Text('Name: ${data['firstName']} ${data['lastName']}`

Reply:
@jmarr445 Ok, it looks like it was just unnecessary to have the extra ' ' so I removed them! Good catch

I definitely think we should refactor this, and make some sort of template. I think I'll create a branch that focuses on refactoring redundant code, but will leave that for next week.